ras_party/controllers/account_controller.py

```diff
@@ -526,3 +556,45 @@ def post_case_event(case_id, party_id, category='Default category message', desc
     logger.info('Case service responded with', status=response.status_code)
     response.raise_for_status()
     return response.json()
+
+
+def request_cases_for_respondent(respondent_id):
+    case_url = f'{current_app.config["RAS_CASE_SERVICE"]}/cases/partyid/{respondent_id}'
+    logger.info('GET', url=case_url)
```

Comment:
The logs are going to get pretty cluttered with INFO's logging the various calls.

Reply:
I'll change some of this to debug and improve the logging a bit, these messages aren't that useful as they are now